[GTK][WPE] WTR: shouldDumpJSConsoleLogInStdErr doesn't work for ports using web-platform.test instead of localhost

carlosgcampos opened the pull request:
#### f0622d73739e1852547b31c5260a32b55d356109
<pre>
[GTK][WPE] WTR: shouldDumpJSConsoleLogInStdErr doesn&apos;t work for ports using web-platform.test instead of localhost
<a href="https://bugs.webkit.org/show_bug.cgi?id=256632">https://bugs.webkit.org/show_bug.cgi?id=256632</a>

Reviewed by Youenn Fablet.

For GTK and WPE shouldDumpJSConsoleLogInStdErr always returns false,
because it compares the test URL assuming localhost is used. We can
remove platform specific expectations where the only difference is the
console messages output.

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/beacon/beacon-cors.https.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/client-hint-request-headers-2.tentative-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/client-hint-request-headers-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/origin-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/request-headers-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/integrity.sub.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/mode-same-origin.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/mode-same-origin.any.worker-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-about.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-blob.sub.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-data.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-others.sub.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-basic.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-basic.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-multiple-origins.sub.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-multiple-origins.sub.any.worker-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-preflight-response-validation.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/data-url-iframe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/data-url-worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/sandboxed-iframe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/policies/csp-blocked-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-schemes.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-schemes.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-to-dataurl.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-to-dataurl.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/request/request-bad-port.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/request/request-bad-port.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch-in-iframe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.https.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.https.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/iframe-loads-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/image-loads-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/scheme-restriction.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/scheme-restriction.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/scheme-restriction.https.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/script-loads-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/data-urls/base64.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/data-urls/processing.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/images/canvas-remote-read-remote-image-redirect-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/metadata/paint-worklet.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/nosniff/parsing-nosniff.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/redirects/data.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/security/redirect-to-url-with-credentials.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/controller-with-no-fetch-event-handler.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-canvas-tainting-image.https-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-cors-xhr.https-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-csp.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-error.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-handled.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-network-error.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-redirect.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-referrer-policy.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-respond-with-argument.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-respond-with-response-body-with-invalid-chunk.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-frame-resource.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-mixed-content-to-inscope.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-mixed-content-to-outscope.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-css-cross-origin.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-fallback.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-redirect.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-response-taint.https-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/invalid-header.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/opaque-response-preloaded.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/opaque-script.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/redirected-response.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/registration-security-error.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/update-recovery.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/worker-interception.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-asciiSep-protocol-string.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-blocked-port.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-invalid-urls.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-non-absolute-url.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-nonAscii-protocol-string.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-protocol-with-space.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-protocols-repeated.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-url-with-space.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-valid-url-protocol-empty.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-valid-url-protocol-empty.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-wrong-scheme.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/basic-auth.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/basic-auth.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/002-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/004-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/007-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/021-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-deleting-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/close/close-multiple-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/close/close-nested-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/close/close-return-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/events/019-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/events/020-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/readyState/003-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/remove-own-iframe-during-onerror.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/security/002-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-denied-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-get-fail-non-simple-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-non-cors-safelisted-content-type-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-post-with-non-cors-safelisted-content-type-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-preflight-denied-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-preflight-request-invalid-status-301-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-preflight-request-invalid-status-400-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-preflight-request-invalid-status-501-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-recursive-failed-request-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-sandboxed-iframe-denied-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-sandboxed-iframe-denied-without-wildcard-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/getresponseheader-error-state-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/open-method-responsetype-set-sync-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/responsetype.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/security-consideration.sub-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-authentication-basic-cors-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-authentication-basic-cors-not-enabled-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-authentication-cors-setrequestheader-no-cred-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-data-arraybuffer.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-non-same-origin-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-redirect-infinite-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-redirect-infinite-sync-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/status-error-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/thrown-error-in-events-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/timeout-sync-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/xmlhttprequest-network-error-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/xmlhttprequest-network-error-sync-expected.txt: Removed.
* Tools/TestRunnerShared/TestFeatures.cpp:
(WTR::shouldDumpJSConsoleLogInStdErr):

Canonical link: <a href="https://commits.webkit.org/269843@main">https://commits.webkit.org/269843@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e037e01f7072e240f1679a84e8c725f92ed31285

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/23644 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/1760 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/24765 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/25803 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/21827 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/23913 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/3340 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/24180 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/22390 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/23888 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/1312 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/20459 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/26399 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/1100 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/21365 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/27637 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/21568 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/21634 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/25400 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/1072 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/18774 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/1051 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5682 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/1485 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/1362 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

gsnedders commented on Tools/TestRunnerShared/TestFeatures.cpp:
```diff
@@ -106,15 +107,28 @@ static std::optional<double> overrideDeviceScaleFactorForTest(const std::string&
+        || path.contains("/streams/piping"_s)
+        || path.contains("/xhr"_s)
+        || path.contains("/webrtc"_s)
+        || path.contains("/websockets"_s);
```
Can we not get away with just having in TestExpectations:

```
imported/w3c/web-platform-tests/beacon [ DumpJSConsoleLogInStdErr ]
```

etc.? Doesn't the inheritance that TestExpectations does work here?

mcatanzaro replied:
I don't think we should use TestExpectations to control test behavior. Let's leave it for documenting whether a test is broken or not.

annevk replied:
Well, we do support `DumpJSConsoleLogInStdErr` there already. I'd rather we configure it in fewer places than more and having test behavior documented in `Tools/` does seem suboptimal.

youennf replied:
We could probably update the inheritance but it does not work this way yet unfortunately, say you have:
```
imported/w3c/web-platform-tests/websockets [DumpJSConsoleLogInStdErr]
imported/w3c/web-platform-tests/websockets/test.html [Pass Failure]
```

test.html will not have the DumpJSConsoleLogInStdErr flag on.

annevk replied:
Ah I was wondering about that later on. That sucks.

gsnedders replied:
Ah, I thought the inheritance should make that work. That seems like something we should file a separate bug for.

annevk replied:
https://bugs.webkit.org/show_bug.cgi?id=256895

youennf reviewed (approved):
LGTM, patch is an improvement and simplifies things.
We can try to remove shouldDumpJSConsoleLogInStdErr as a follow-up, this patch helps this by removing some expected.txt files.

carlosgcampos committed:
[GTK][WPE] WTR: shouldDumpJSConsoleLogInStdErr doesn't work for ports using web-platform.test instead of localhost
https://bugs.webkit.org/show_bug.cgi?id=256632

Reviewed by Youenn Fablet.

For GTK and WPE shouldDumpJSConsoleLogInStdErr always returns false,
because it compares the test URL assuming localhost is used. We can
remove platform specific expectations where the only difference is the
console messages output.

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/beacon/beacon-cors.https.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/client-hint-request-headers-2.tentative-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/client-hint-request-headers-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/origin-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/cors/request-headers-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/integrity.sub.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/mode-same-origin.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/mode-same-origin.any.worker-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-about.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-blob.sub.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-data.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/basic/scheme-others.sub.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-basic.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-basic.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-multiple-origins.sub.any-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-multiple-origins.sub.any.worker-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/cors-preflight-response-validation.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/data-url-iframe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/data-url-worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/cors/sandboxed-iframe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/policies/csp-blocked-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-schemes.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-schemes.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-to-dataurl.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/redirect/redirect-to-dataurl.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/request/request-bad-port.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/api/request/request-bad-port.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch-in-iframe-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.https.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/fetch.https.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/iframe-loads-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/image-loads-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/scheme-restriction.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/scheme-restriction.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/scheme-restriction.https.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/cross-origin-resource-policy/script-loads-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/data-urls/base64.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/data-urls/processing.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/images/canvas-remote-read-remote-image-redirect-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/metadata/paint-worklet.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/nosniff/parsing-nosniff.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/redirects/data.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/fetch/security/redirect-to-url-with-credentials.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/controller-with-no-fetch-event-handler.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-canvas-tainting-image.https-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-cors-xhr.https-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-csp.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-error.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-handled.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-network-error.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-redirect.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-referrer-policy.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-respond-with-argument.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event-respond-with-response-body-with-invalid-chunk.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-frame-resource.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-mixed-content-to-inscope.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-mixed-content-to-outscope.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-css-cross-origin.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-fallback.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-redirect.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-response-taint.https-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/invalid-header.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/opaque-response-preloaded.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/opaque-script.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/redirected-response.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/registration-security-error.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/update-recovery.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/worker-interception.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-asciiSep-protocol-string.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-blocked-port.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-invalid-urls.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-non-absolute-url.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-nonAscii-protocol-string.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-protocol-with-space.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-protocols-repeated.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-url-with-space.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-valid-url-protocol-empty.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-valid-url-protocol-empty.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/Create-wrong-scheme.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/basic-auth.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/basic-auth.any.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/002-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/004-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/007-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/constructor/021-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-deleting-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/close/close-multiple-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/close/close-nested-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/close/close-return-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/events/019-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/events/020-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/readyState/003-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/remove-own-iframe-during-onerror.window-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/websockets/security/002-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-denied-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-get-fail-non-simple-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-non-cors-safelisted-content-type-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-post-with-non-cors-safelisted-content-type-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-basic-preflight-denied-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-preflight-request-invalid-status-301-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-preflight-request-invalid-status-400-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-preflight-request-invalid-status-501-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-recursive-failed-request-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-sandboxed-iframe-denied-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/access-control-sandboxed-iframe-denied-without-wildcard-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/getresponseheader-error-state-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/open-method-responsetype-set-sync-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/responsetype.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/security-consideration.sub-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-authentication-basic-cors-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-authentication-basic-cors-not-enabled-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-authentication-cors-setrequestheader-no-cred-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-data-arraybuffer.any-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-non-same-origin-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-redirect-infinite-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/send-redirect-infinite-sync-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/status-error-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/thrown-error-in-events-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/timeout-sync-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/xmlhttprequest-network-error-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/xhr/xmlhttprequest-network-error-sync-expected.txt: Removed.
* Tools/TestRunnerShared/TestFeatures.cpp:
(WTR::shouldDumpJSConsoleLogInStdErr):

Canonical link: https://commits.webkit.org/269843@main
